playbooks/common/openshift-grafana/config.yml

```diff
@@ -0,0 +1,6 @@
+---
+- name: Deploy grafana server
```

Comment:
Not exactly pertinent to this file, but in the `byo` playbook directory you should include a file that calls the standard include (`init/main.yml`) and then your `common` playbook, like this:

https://github.com/openshift/openshift-ansible/blob/master/playbooks/byo/openshift-management/config.yml

Reply:
done